tests.py

```diff
@@ -845,6 +845,34 @@ def test_ignore_bad_ideas_args(self):
         self.assertEqual(check_manifest.IGNORE_BAD_IDEAS,
                          ['x', 'y', 'z'])
 
+    def test_verbose_arg(self):
+        import check_manifest
+        check_manifest.VERBOSE = False
+        sys.argv.append('--verbose')
+        check_manifest.main()
+        self.assertTrue(check_manifest.VERBOSE)
+        check_manifest.VERBOSE = False
```

Comment:
This sort of cleanup is better done in `tearDown()`.  (This way if `self.assertTrue` raises an exception, cleanup will still run.)